opentracing/span.go

```diff
@@ -0,0 +1,165 @@
+package opentracing
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	opentracing "github.com/opentracing/opentracing-go"
+	"sourcegraph.com/sourcegraph/appdash"
+)
+
+// Span is the Appdash implementation of the `opentracing.Span` interface.
```

Comment:
style nit (applies to opentracing-go pkg too): Go doesn't use markdown code blocks, so this would just be `the opentracing.Span interface`